src/theme/css/chrome.css

```diff
@@ -159,11 +159,14 @@ a > .hljs {
 
 .mobile-nav-chapters {
     font-size: 2.5em;
-    text-align: center;
     text-decoration: none;
     width: 90px;
+    height: 55px;
```

Comment:
This height is based on the rendering result on the Chrome browser without these changes. We need to add it because CSS grids seem to ignore `line-height` which were previously respected.